### PR TITLE
{phys}[system/system] COMSOL v6.2.0.290

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
- [x] depends on #https://github.com/easybuilders/easybuild-easyblocks/pull/3044

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/f8904ca4df16404935a8d929983e7038 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3044
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/b4348b74689e77114febfa810619abd3 for a full test report.

File: easybuild/easyconfigs/c/COMSOL/COMSOL-6.2.0.290.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Note: sources from COMSOL are often named poorly. You can view the exact version in the "ver" file inside the iso:
+# 7z x COMSOL-*.iso ver; cat ver
+sources = ['COMSOL-%(version)s.iso']
+checksums = ['a90744cb29379a871ca53a0f525a0c378155f5facb3cb5e4dd540ccfc7f4eeec']
+
+download_instructions = 'Obtain from comsol.se'
+
+osdependencies = [('p7zip-plugins', 'p7zip-full'), 'csh']  # for extracting iso-files
+
+# license_file = 'license.dat'  # or EB_COMSOL_LICENSE_FILE
```
Comment (smoors) [inline easybuild/easyconfigs/c/COMSOL/COMSOL-6.2.0.290.eb:22]:
```suggestion
# license_file = 'license.dat'  # or EB_COMSOL_LICENSE_FILE or LMCOMSOL_LICENSE_FILE or LM_LICENSE_FILE
```

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/1b3f45b52107e3835352ba82a8684537 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/2d1d95cd927e1718a0035e1ccc2db37e for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Failed during parsing of the easyconfigs, so no ecs were built (1 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/306274571f11aebc660ea1008071d374 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Failed during parsing of the easyconfigs, so no ecs were built (1 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/20881320380af9daef199c466069b7d0 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/8129a44119e1ab403c1a6260c05bfce3 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @Micket!

File: easybuild/easyconfigs/c/COMSOL/COMSOL-6.2.0.290.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/c/COMSOL/COMSOL-6.2.0.290.eb]: Erh.. why would we actively encourage the use of the old variable name. I think this addition is just worse.

**Pull request merged**